tools/codegen/dest/register_dispatch_key.py

```diff
@@ -461,6 +489,16 @@ def gen_class_set_output_body(self, k: SchemaKind) -> str:
         else:
             maybe_set_guard_line = maybe_set_guard = ''
 
+        if maybe_create_proxy:
+            create_proxy = """
+auto maybe_proxy = maybe_create_proxy(out, sizes, strides, options);
+if (maybe_proxy.has_value()) {
```

Comment:
Nit. This might better be around a `C10_UNLIKELY`? The user is barely ever going to provide an out with the wrong strides.